enterprise-suite/console_test/util/monitor/monitor.go

```diff
@@ -0,0 +1,57 @@
+package monitor
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+)
+
+// Connection to es-monitor-api server
+type Connection struct {
+	url string
+}
+
+// Make adds a simple threshold metric with non-configurable parameters.
+// TODO: Make this more configurable!
+func (m *Connection) Make(name string, metric string) error {
```

Comment:
The function name confused me.  Would it make sense to be "MakeMonitor" or something like that?

Reply:
Good suggestion, renamed to MakeMonitor